helios-core/src/test/kotlin/helios/instances/InstancesArrowTest.kt

```diff
@@ -0,0 +1,32 @@
+package helios.instances
+
+import arrow.core.Either
+import arrow.core.Right
+import arrow.data.NonEmptyList
+import arrow.test.UnitSpec
+import arrow.test.generators.nonEmptyList
+import helios.core.Json
+import helios.test.generators.alphaStr
+import io.kotlintest.assertions.arrow.either.beLeft
+import io.kotlintest.properties.Gen
+import io.kotlintest.properties.forAll
+import io.kotlintest.should
+
+class InstancesArrowTest : UnitSpec() {
+
+    init {
+
+        "NonEmptyList should be encode and decode successfully" {
+            forAll(Gen.nonEmptyList(Gen.alphaStr())) { list ->
```

Comment:
using `assertAll` instead of `forAll` you should be able to use `shouldBeRight(list)`